eng/pipelines/templates/steps/vmr-pull-updates.yml

```diff
@@ -17,6 +17,11 @@ parameters:
   type: string
   default: $(Agent.BuildDirectory)/vmr
 
+- name: architecture
+  displayName: Architecture used for build artifacts
```

Comment:
soo this is `targetArchitecture`? 😄

Reply:
![image](https://github.com/dotnet/installer/assets/308992/692ac60f-85d8-4ae9-ad49-0c6f1a80ff4c)

It's consumed here https://github.com/dotnet/installer/blob/main/eng/pipelines/templates/jobs/vmr-build.yml#L398 which uses artifact